aten/src/ATen/SparseTensorImpl.h

```diff
@@ -56,7 +56,8 @@ struct CAFFE2_API SparseTensorImpl : public TensorImpl {
   // respect to indices and values
   void raw_resize_(int64_t sparse_dim, int64_t dense_dim, IntArrayRef size) {
     TORCH_CHECK(allow_tensor_metadata_change(), "raw_resize_ ", err_msg_tensor_metadata_change_not_allowed);
-    sizes_ = size.vec();
+    sizes_and_strides_.resize(size.size());
+    std::copy(size.begin(), size.end(), sizes_and_strides_.sizes_begin());
```

Comment:
I'd probably also vote for a helper method for abstracting over `std::copy` calls

Reply:
adding set_sizes, but it feels a little dangerous to me to cover up a `resize` like that.